components-core/src/main/java/com/adyen/checkout/components/core/internal/data/model/AnalyticsSetupRequest.kt

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by josephj on 4/7/2023.
+ */
+
+package com.adyen.checkout.components.core.internal.data.model
+
+import com.adyen.checkout.components.core.Amount
+import com.adyen.checkout.core.exception.ModelSerializationException
+import com.adyen.checkout.core.internal.data.model.ModelObject
+import com.adyen.checkout.core.internal.data.model.ModelUtils
+import com.adyen.checkout.core.internal.data.model.getLongOrNull
+import com.adyen.checkout.core.internal.data.model.getStringOrNull
+import com.adyen.checkout.core.internal.data.model.optStringList
+import kotlinx.parcelize.Parcelize
+import org.json.JSONException
+import org.json.JSONObject
+
+@Parcelize
+internal data class AnalyticsSetupRequest(
+    val version: String?,
+    val channel: String?,
+    val platform: String?,
+    val locale: String?,
+    val component: String?,
+    val flavor: String?,
+    val deviceBrand: String?,
+    val deviceModel: String?,
+    val referrer: String?,
+    val systemVersion: String?,
+    val containerWidth: Long?,
```

Comment:
Do you think this will ever be bigger than `Int.MAX_VALUE`?

Reply:
Definitely not, but I was just mirroring the types on the backend

Reply:
Ohh that makes sense 👍